x/nft/migrations/v2/keys.go

```diff
@@ -0,0 +1,106 @@
+package v2
+
+import (
+	"bytes"
+
+	"cosmossdk.io/x/nft"
+	"cosmossdk.io/x/nft/internal/conv"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/address"
+)
+
+var (
```

Comment:
Could you move these to like a v1 folder?
It seems from other migration that as they exist since v1, we put those variables in v1.